tests/sodalint/LICENSE

```diff
@@ -0,0 +1,64 @@
+CSIRO Open Source Software Licence v1.0
+
```

Comment:
Why are you checking sodalint in?  Why not just download it for the test or as part of the setup, or put it in the container?  By checking in big files like this to the repo, it can slow you down as they never really get deleted.

Reply:
This comment applies to the .jar file mostly, but you also wouldn't have to check this in.

Reply:
The .jar file is 7MB, which is argubably no longer considered big today (LICENSE file added just in case).  sodalint is lightly used out there; other than GitHub, there is no public repo for its release binaries.  I'm aware of 'git checkout tags/thetagoftherelease', which can apply to sodalint here.  But once I learn how that's handled exactly in our Jenkins-CI env, it's something to be considered.

Reply:
At the very least, adding third party code to a repo should be a separate commit, such that it is easy to revert later on if you change the way you do the sodalint testing.

Reply:
agree.